apps/meeting/src/app.tsx

```diff
@@ -31,24 +31,7 @@ const App: FC = () => (
           <Notifications />
           <ErrorProvider>
             <VoiceFocusProvider>
-              <MeetingProvider {...meetingConfig}>
-                <NavigationProvider>
-                  <Switch>
-                    <Route exact path={routes.HOME} component={Home} />
-                    <Route path={routes.DEVICE}>
-                      <NoMeetingRedirect>
-                        <DeviceSetup />
-                      </NoMeetingRedirect>
-                    </Route>
-                    <Route path={routes.MEETING}>
-                      <NoMeetingRedirect>
-                        <MeetingModeSelector />
-                      </NoMeetingRedirect>
-                    </Route>
-                  </Switch>
-                </NavigationProvider>
-                <MeetingEventObserver />
-              </MeetingProvider>
+              <MeetingWrapper />
```

Comment:
Can we add an option here:
```
{ isWebAudioEnabled ? <MeetingWrapperWithVF /> : <MeetingWrapper /> }
```
And`<MeetingWrapperWithVF />` will return the children wrapped within `VoiceFocusProvider`, while `<MeetingWrapper />` does not have `VoiceFocusProvider` inside.

Reply:
Actually VoiceFoucusProvider is imported from component library so we cannot change it but i have moved VoiceFocusProvider in MeetingWrapper.

Reply:
@jing-chen1  I tried to remove VoiceFocusProvider but AudioInputProvider is using useVoiceFocus hook so we need to wrap the components with VoiceFocusProvider. Otherwise getting the below error 

index.tsx:184 Uncaught Error: useVoiceFocus must be used within VoiceFocusProvider
    at useVoiceFocus (localhost/:81253)
    at AudioInputProvider (localhost/:81269)
    at renderWithHooks (localhost/:55974)
    at mountIndeterminateComponent (localhost/:58800)
    at beginWork (localhost/:60038)
    at HTMLUnknownElement.callCallback (localhost/:44934)
    at Object.invokeGuardedCallbackDev (localhost/:44983)
    at invokeGuardedCallback (localhost/:45045)
    at beginWork$1 (localhost/:64948)
    at performUnitOfWork (localhost/:63760)